content/monad-transformers.md

```diff
@@ -23,13 +23,57 @@ A `Reader r a` in which the resulting `a` is wrapped in any other monad, i.e. `r
 
 ### StateT
 
-A `State s a` in which the return value `a` and state `s` are wrapped in any other monad, i.e. `s -> m (a, s)`
+A `State s a` in which the return value and state, namely `(a, s)`, are wrapped in any other monad, i.e. `s -> m (a, s)`
 
 ### EitherT
 
 An `Either e a` wrapped in any other monad, i.e. `m (Either e a)`
 
-* Simple examples of usage
+## Simple examples of usage
+
+### MonadTrans
+
+[transformers](https://www.stackage.org/lts-3.20/package/transformers-0.4.2.0) is a widely used package which provides transformer versions of various monads. 
```

Comment:
To show the most up-to-date version of the docs, I'd recommend making the link point to:

https://www.stackage.org/package/transformers